### PR TITLE
Repackage - Multi Cloud Attack & Network Threat PE

### PR DESCRIPTION
**Pull request by v-rusraut**
Required items, please complete
   
   Change(s):
   - Repackaging

   Reason for Change(s):
   - Tagged for dependent solutions for deployment

   Version Updated:
   - Network Threat Protection Essentials (3.0.1)
   - Multi Cloud Attack Coverage Essentials - Resource Abuse (3.0.1)

   Testing Completed:
   - Done
   Network Threat Protection Essentials
NetworkTP
Multi Cloud Attack Coverage Essentials 
MultiCloud

### CODE REVIEW

File: Solutions/Network Threat Protection Essentials/Hunting Queries/UseragentExploitPentest.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 id: df75ac6c-7b0b-40d2-82e4-191c012f1a07
 name: Exploit and Pentest Framework User Agent
 description: |
+    'This query detects suspicious user agent strings used by exploit and pen test frameworks. It's based on sigma rules and focuses on data sources containing the UserAgent field.'
```
Comment (rahul0216) [inline Solutions/Network Threat Protection Essentials/Hunting Queries/UseragentExploitPentest.yaml:4]: This query detects suspicious user agent strings used by exploit and pen test frameworks. (This is good enough)

**Review by rahul0216 — COMMENTED** [review]
Mostly looks good. Please address inline comment.

File: Solutions/Network Threat Protection Essentials/Hunting Queries/UseragentExploitPentest.yaml (hunk 1 above)
Comment (v-rusraut) [reply Solutions/Network Threat Protection Essentials/Hunting Queries/UseragentExploitPentest.yaml]: Resolved

**Review by rahul0216 — DISMISSED** [review]
Hunting query descriptions looks good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**